addons/purchase/views/portal_templates.xml

```diff
@@ -156,8 +156,8 @@
                               <div class="small mb-1"><strong class="text-muted">Purchase Representative</strong></div>
                               <div class="row flex-nowrap">
                                   <div class="col flex-grow-0 pr-2">
-                                      <img class="rounded-circle mr4 float-left o_portal_contact_img" t-if="order.user_id.image_1024" t-att-src="image_data_uri(order.user_id.image_1024)" alt="Contact"/>
-                                      <img class="rounded-circle mr4 float-left o_portal_contact_img" t-if="not order.user_id.image_1024" src="/web/static/img/placeholder.png" alt="Contact"/>
+                                      <img class="rounded-circle mr4 float-left o_portal_contact_img" t-if="order.user_id.avatar_1024" t-att-src="image_data_uri(order.user_id.avatar_1024)" alt="Contact"/>
+                                      <img class="rounded-circle mr4 float-left o_portal_contact_img" t-if="not order.user_id.avatar_1024" src="/web/static/img/placeholder.png" alt="Contact"/>
```

Comment:
Is it possible that the avatar is not set? This condition is for a placeholder that we probably don't need anymore.